releasenotes/config.yaml

```
@@ -56,6 +56,11 @@ template: |
     - |
       For new features such as a new integration or component. Use present tense with the following format:
       Format: <scope>: This introduces <new_feature_or_component>.
+  issues:
+    - |
+      For known issues. Use present tense with the following format:
+      Format: <scope>: There is currently a known <symptom_of_issue> issue with <affected_code>.
```

currently is implied (but I am the last person who should be giving writing tips, feel free to resolve)

```suggestion
      Format: <scope>: There is a known <symptom_of_issue> issue with <affected_code>.
```